scripts_test/config/end_of_training_callbacks_config.yaml

```diff
@@ -2,7 +2,7 @@
     {"class":"WriteToDbCallback",
      "kwargs":{"db_path": "runs.db"}},
     {"class":"EmailCallback",
-     "kwargs":{"to_emails":["avanti@stanford.edu"],
+     "kwargs":{"to_emails":["therajiv@gmail.com"],
```

Comment:
Might want to leave that as my email in the committed version unless you want to get random emails from people using the framework for the first time :-)